crates/ark/src/lsp/help.rs

```diff
@@ -32,7 +34,31 @@ pub enum Status {
 }
 
 impl RHtmlHelp {
-    pub unsafe fn new(topic: &str, package: Option<&str>) -> Result<Option<Self>> {
+    pub unsafe fn from_topic(topic: &str, package: Option<&str>) -> anyhow::Result<Option<Self>> {
```

Comment:
Why is this unsafe? Because it calls R? Can we use some other signal like `r_` (if specific methods) or `r::` (if the whole struct)?